source/tutorials/manage-minio-tenant.rst

```diff
@@ -17,14 +17,245 @@ MinIO Tenant.
 Expand a MinIO Tenant
 ---------------------
 
-This procedure documents expanding a MinIO Tenant.
+This procedure documents expanding a MinIO Tenant by adding a new 
+Pool of MinIO Pods and storage resources.
+
```

Comment:
once the Operator Console is ready for prime-time, this page will *also* include a procedure specific to using that tool vs kubectl minio.